### DISCUSSION TITLE
反馈一个bug

### DISCUSSION THREAD

**Discussion by hao8497** [amll-dev/applemusic-like-lyrics#237] [General]
在新版网易云中进度条在切换音乐后不归零
例如音乐播放到3分30秒，我切换到下一首音乐，音乐从头播放，但是进度条会从3分30秒继续移动，导致歌词时间对不上，进度条走完但歌曲未播完

**Comment by godd221** [comment]
试试关闭   杂项（使用插值平滑播放进度）

**Reply by hao8497** [reply]
OK了，谢谢
